README.md

```diff
@@ -1 +1,50 @@
 # Bigcommerce Sample App: Hello World in Ruby and Sinatra
+
+## App Registration
+ - Create a trial shop on [Bigcommerce](https://www.bigcommerce.com/)
+ - Go to the [developer portal](https://developer.bigcommerce.com/) and log in
+ - Go to "My Apps" (linked in the navbar)
+ - Click the button "Create an app", enter a name for the new app and then click the button "Create"
+ - Go right to section 4 (Technical) in the app dialog and enter the following URLs:
+   - *Auth Callback URL* - `https://<app hostname>/auth/bigcommerce/callback`
+   - *Load Callback URL* - `https://<app hostname>/load`
+   - *Uninstall Callback URL* - `https://<app hostname>/uninstall`
+ - Enable the following permission scopes under *OAuth scopes*:
+   - *Orders* - MODIFY
```

Comment:
This sample app only makes use of the "products read only" scope (in the `/` route, where we use the API to pull the store's products. Unless the app is updated to make use of these additional scopes, I think they should be removed from the README.